docs/docs/en/guide/project/workflow-definition.md

```diff
@@ -97,24 +97,47 @@ The following are the operation functions of the workflow definition list:
-    * You can select complement time range (When the scheduled configuration is not online, the daily complement will be performed by default according to the selected time range.If the timing configuration is online, it will be complemented according to the selected time range in combination with the timing configuration) when executing a timing workflow definition. For example, need to fill in the data from 1st May to 10th May, as shown in the figure below:
-
-    ![workflow-date](../../../../img/new_ui/dev/project/workflow-date.png)
-
-    > Serial mode: the complement execute sequentially from 9th May to 10th May, and the process instance page generates 2 process instances;
-
-    > Parallel mode: The tasks from 9th May to 10th May execute simultaneously, and the process instance page generates 2 process instances;
- 
-    * You can manually enter the complement dates (independent from the timing configuration) to execute the complement data to workflow definitions. The figure below is an example to make complement data from 9th May to 10th May:
- 
-    ![workflow-date](../../../../img/new_ui/dev/project/workflow_date_manual.png)
-
-    > Serial mode: the complement execute sequentially from 9th May to 10th May, and the process instance page generates 2 process instances;
-
-    > Parallel mode: The tasks from 9th May to 10th May execute simultaneously, and the process instance page generates 2 process instances;
-
+  * Complement: refers to running the workflow definition within the specified date range and generating the corresponding workflow instance according to the complement policy. The complement policy includes two modes: serial complement and parallel complement. The date can be selected on the page or entered manually.
+  
+    * Serial complement: within the specified time range, complement is executed from the start date to the end date, and multiple process instances are generated in turn; Click Run workflow and select the serial complement mode: for example, from July 9 to July 10, execute in sequence, and generate two process instances in sequence on the process instance page.
+        
+        ![workflow-serial](../../../../img/new_ui/dev/project/workflow-serial.png)
+    
+    * Parallel Replenishment: within the specified time range, replenishment is performed simultaneously for multiple days, and multiple process instances are generated at the same time. Enter date manually: manually enter a date in the comma separated date format of 'yyyy MM DD hh:mm:ss'.Click Run workflow and select the parallel complement mode: for example, execute the tasks from July 9 to July 10 at the same time, and generate two process instances on the process instance page at the same time.
```

Comment:
'execute the tasks from July 9 to July 10 at the same time',   replace the tasks with workflow definition.